docs/products/postgresql/howto/migrate-aiven-db-migrate.rst

```diff
@@ -1,32 +1,38 @@
-Migrate to Aiven for PostgreSQL with ``aiven-db-migrate``
-=========================================================
+Migrating to Aiven for PostgreSQL with ``aiven-db-migrate``
+===========================================================
 
-This article describes how to migrate a PostgreSQL cluster from an external source to an Aiven for PostgreSQL service using the ``aiven-db-migrate`` tool (the project is open source, find it `on GitHub <https://github.com/aiven/aiven-db-migrate>`_). To learn more, please visit :doc:`the overview of the process<../concepts/aiven-db-migrate>`.
+The ``aiven-db-migrate`` tool is an open source project available on `GitHub <https://github.com/aiven/aiven-db-migrate>`_), and it is the preferred way to perform the migration. 
 
-Variables
-'''''''''
+``aiven-db-migrate`` performs a schema dump and migration first to ensure schema compatibility.
 
-Replace these placeholder variables in the code samples below:
+It supports both logical replication, and using a dump and restore process. 
+Logical replication is the default method which keeps the two databases synchronized until the replication is interrupted. 
+If the preconditions for logical replication are not met for a database, the migration falls back to using ``pg_dump``.
 
-==================      =======================================================================
-Variable                Description
-==================      =======================================================================
-``SRC_HOSTNAME``        Hostname for source PostgreSQL connection
-``SRC_PORT``            Port for source PostgreSQL connection
-``SRC_DATABASE``        Database Name for source PostgreSQL connection
-``SRC_USERNAME``        Username for source PostgreSQL connection
-``SRC_PASSWORD``        Password for source PostgreSQL connection
-``SRC_SSL``             SSL setting for source PostgreSQL connection
-``DEST_PG_NAME``        Name of the destination Aiven PostgreSQL service
-``DEST_PG_PLAN``        Aiven Plan for the destination Aiven PostgreSQL service
-==================      =======================================================================
+.. Note::
+    You can use logical replication when migrating from AWS RDS PostgreSQL 10+, whereas the Google Cloud Platform's PostgreSQL for CloudSQL does not support it.
+
+.. Warning::
```

Comment:
Is there a way we can separate or otherwise re-organise these admonitions? They look a bit strange next to one another.

Reply:
It's good to put the warning before the risky steps though!! I just spotted where you moved this from and facepalmed

Reply:
I can move it directly before the "actual procedure". That will solve it.